__e2e__/peruse.spec.js

```diff
@@ -242,8 +244,84 @@ describe( 'main window', () =>
         expect( bookmarks ).toMatch( 'shouldappearinbookmarks' );
 
     } );
+    it( 'can check if settings menu exists', async () =>
+    {
+        expect.assertions( 1 );
+        const { client } = app;
+        await delay( 4500 );
+
+        await setClientToMainBrowserWindow( app );
+        await delay( 4500 );
+        const menuExists = await client.waitForExist( BROWSER_UI.SETTINGS_MENU__BUTTON, WAIT_FOR_EXIST_TIMEOUT );
+        expect( menuExists ).toBeTruthy();
+    } );
 
+    it( 'can open settings menu', async () =>
+    {
+        expect.assertions( 1 );
+        const { client } = app;
+        await delay( 4500 );
 
+        await setClientToMainBrowserWindow( app );
+        await delay( 4500 );
+        await client.click( BROWSER_UI.SETTINGS_MENU__BUTTON );
+        await delay( 4500 );
+        const settings = await client.waitForExist( BROWSER_UI.SETTINGS_MENU, WAIT_FOR_EXIST_TIMEOUT );
```

Comment:
similar to how you have `menuExists` above, lets make sure the variable naming that we're checking is clear.

So `settingsMenuIsShown` for eg here, (but also check throughout).

Reply:
I have renamed the variable names appropriately.